code/modules/reagents/reagent_containers/cups/glassbottle.dm

```diff
@@ -151,6 +151,9 @@
 	if(!isGlass)
 		return
 
+	if(QDELETED(target))
+		return
```

Comment:
Arguably shouldn't have a qdeleted check - otherwise destroying an object with a bottle wouldn't break said bottle

Reply:
I spose everything but the smash() part could be wrapped in the block since that's the problematic part